contributors/devel/cri-testing-policy.md

```diff
@@ -50,16 +50,16 @@ To publish tests results, please submit a proposal in the
 briefly explaining your runtime, providing at least two maintainers, and
 assigning the proposal to the leads of SIG-Node.
 
-These test results should be published under the `sig-node` tab, grouped by the
-runtimes, organized as follows.
+These test results should be published under the `sig-node` tab, organized
+as follows.
 
 ```
-sig-node -> sig-node-{Kubernetes-version} -> sig-node-{runtime-name} -> [page containing all test jobs]
-```
+sig-node -> sig-node-cri-{Kubernetes-version} -> [page containing the required jobs]
```

Comment:
it's unclear to me what 'required' means here, maybe a sample list of required jobs?

Reply:
This is described in the previous section: node conformance & node feature suite.